.ci/static-checks.sh

```diff
@@ -429,6 +429,71 @@ check_docs()
 	rm -f "$url_map" "$invalid_urls"
 }
 
+# Tests to apply to all files.
+#
+# Currently just looks for TODO/FIXME comments that should be removed.
```

Comment:
s/removed/converted to or annotated with an Issue/ ?

Reply:
Good call - fixed.